api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
Yup, [here it is](https://github.com/CenterForOpenScience/osf.io/blob/eac466dba48543c997dbcbecc4ed2955e41ae23e/website/static/js/comment.js#L514-L532).

![gone2jppd7](https://cloud.githubusercontent.com/assets/134455/14959395/2b509396-105d-11e6-9cd6-89297987e821.gif)